sycl/test/basic_tests/image_accessor_readsampler.cpp

```diff
@@ -22,17 +21,22 @@ namespace s = cl::sycl;
 
 template <int unique_number> class kernel_class;
 
-void validateReadData(s::cl_float4 ReadData, s::cl_float4 ExpectedColor) {
-  // Maximum difference of 1.5 ULP is allowed.
+void validateReadData(s::cl_float4 ReadData, s::cl_float4 ExpectedColor,
+                      s::cl_int precision = 1) {
+  // Maximum difference of 1.5 ULP is allowed when precision = 1.
   s::cl_int4 PixelDataInt = ReadData.template as<s::cl_int4>();
   s::cl_int4 ExpectedDataInt = ExpectedColor.template as<s::cl_int4>();
   s::cl_int4 Diff = ExpectedDataInt - PixelDataInt;
 #if DEBUG_OUTPUT
```

Comment:
do we really need this output?

Reply:
It is used for debugging mostly. Won't this be needed if some change break this test?

Reply:
Usually @bader against such debug outputs.